### PR TITLE
Zaps, socket reuse, embedded links

### PR DESCRIPTION
**Pull request by brugeman**

### CODE REVIEW

File: src/components/meta.js [hunk 1]
```diff
@@ -20,27 +28,33 @@ function Meta({ note, repliesCount, repostsCount, likesCount }) {
...
+    formattedZapAmount = formatZapAmount(zapAmount);
   }
 
   return (
     <div class="cardMeta">
       <div class="cardDate">{formattedDate}</div>
       <hr />
       <div class="cardInteractions">
-        <div class="interactionContainer">
+        <div class="interactionContainer" title="Amount of Zaps">
```
Comment (erskingardner) [inline src/components/meta.js]: I'd change the title here to "Total sats zapped"

File: index.html [hunk 2]
```diff
@@ -11,7 +11,8 @@
   <body>
     <h1>Testing the Nostr Embed</h1>
 
-    <div id="nostr-embed"></div>
+    <div id="nostr-embed-1"></div>
+    <div id="nostr-embed-2"></div>
```
Comment (erskingardner) [inline index.html:15]: We should probably add some to the readme to explain how embedding works. I'm happy to do that after this PR is merged though.

**Review by erskingardner — COMMENTED** [review]
One small comment on copy, otherwise looks good.

File: src/components/meta.js (hunk 1 above)
Comment (brugeman) [reply src/components/meta.js]: Agree

File: index.html (hunk 2 above)
Comment (brugeman) [reply index.html:15]: Ok

**Pull request merged**